chevah_build

```diff
@@ -59,6 +67,8 @@ TIMESTAMP=`date +'%Y%m%d'`
 # "ppc" instead of "ppc64".
 # We also use $ARCH when building the statically compiled libffi and GMP.
 export ARCH
+# We want to check if OS is archlinux when installing libffi
```

Comment:
I think that not having this comment, and reusing the previous one should be fine